packages/adblocker-electron-example/index.js

```diff
@@ -0,0 +1,2 @@
+require('ts-node').register();
```

Comment:
:heart:

Reply:
Is there something I should do about this?

Reply:
No, sorry for the confusion. I just found it very nice!